src/ESM.sol

```diff
@@ -67,6 +68,11 @@ contract ESM {
         emit Rely(msg.sender);
     }
 
+    modifier live() {
+        require(stopped == 0, "ESM/permanently-disabled");
+        _;
+    }
```

Comment:
since this is a "core" module, we might want to follow the dss pattern of doing requires in the functions rather than a modifer which would alleviate the naming conflict with `live`. (if we do that then `stop` could become `cage` too)